vendor/vendor.json

```diff
@@ -57,6 +57,18 @@
 			"revisionTime": "2016-06-30T22:46:53-07:00"
 		},
 		{
+			"checksumSHA1": "RBwpnMpfQt7Jo7YWrRph0Vwe+f0=",
+			"path": "github.com/coreos/go-systemd/activation",
+			"revision": "d2196463941895ee908e13531a23a39feb9e1243",
```

Comment:
Please update the revision with a real client hash after the client side is merged.

Reply:
yep, that's the "DUMMY COMMIT" that's going to get replaced.